### ISSUE TITLE
Information for get​Item(index) is incomplete / misleading

### ISSUE THREAD

**Issue by cindymeister** [OfficeDev/office-js#438]
The explanation for getItem(index) is: "Gets a content control by its index in the collection." And the expected data type is "number". This implies the method returns the content control at the given position in the document (first, second, etc.) Using the method in this manner returns the error "ItemNotFound: We couldn’t find the item you requested. Check the OfficeExtension.Error.debugInfo for more information." On Stack Overflow Juan B. Labra states: "getItem() works with IDs". The ID value of a content control is of type String (in the COM APIs) and it's also not an "index" value. I lost a lot of time trying to get this to work. Plesae update the documentation.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d6b1d537-d611-5bbd-1f09-017881ea2e50
* Version Independent ID: 0a843707-89d9-1738-5693-a6d4cdf0d008
* Content: [Word.ContentControlCollection class](https://docs.microsoft.com/en-us/javascript/api/word/word.contentcontrolcollection?view=office-js#getitem-index-)
* Content Source: [docs/docs-ref-autogen/word/word.contentcontrolcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/live/docs/docs-ref-autogen/word/word.contentcontrolcollection.yml)
* Product: **office 365**
* GitHub Login: @o365devx
* Microsoft Alias: **o365devx**

**Comment by JuaneloJuanelo** [comment]
@Rick-Kirkham : @cindymeister  is right, the documentation is misleading.  [This one needs to be modified](https://docs.microsoft.com/en-us/javascript/api/word/word.contentcontrolcollection?view=office-js#getitem-index-).

and now to add more details.

1. the getItem methods in Office.js universally gets an item within the collection by ID. This is true in Word and in Excel. IN the documentation we INCORRECTLY say its by Index.
2. The getItemAt methods in Office.js are the ones used for getting an item by its position in the collection.  Given that in some conditions this is very confusing in word we did not implemented this method for content controls. 
3. In the case of Word it was, effectively, not needed since we already have a getByID method on the control collection, at the time we only added this method to be consistent with the rest of the API.
4. Finally getItem, as Cindy refers DO work with content control IDs, who happen to [ultimately] be numbers. (and read only). They are automatically assigned by Word and cannot be changed programmatically. We handle this different than in the COM API, although there is also a "stringified" number.


Bottom line I am assigning back to Ricky to update the docs! 

Thanks @cindymeister !

-Juan.
